client/src/components/LobbyChat.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+  import Modal from "./Modal.svelte";
+  import { onMount, onDestroy } from "svelte";
+  import { authStore } from "$stores/auth";
+  import { onSnapshot, orderBy, query, where } from "firebase/firestore";
+  import type { LobbyChatMessage, Lobby, Player } from "$lib/firebase/firestore-types/lobby";
+  import type { Unsubscribe, User } from "firebase/auth";
+  import { getLobbyChatCollection } from "$lib/firebase/firestore-collections";
+  import { addLobbyChatMessages } from "$lib/firebase/chat";
+  import ChatMessages from "./ChatMessages.svelte";
+
+  export let lobbyData: Lobby & { id: string };
+
+  let user = $authStore as User;
+  let userInfo: Player;
+  let openSignInModal = false;
+  let errorMessage: string = "";
+  let chatMessages: LobbyChatMessage[] = [];
+  let unsubscribeChatMessages: Unsubscribe | undefined = undefined;
+
+  onMount(async () => {
+    let messageQuery;
+    if (userInfo.alive) {
+      messageQuery = query(
+        getLobbyChatCollection(lobbyData.id),
+        where("alive", "==", true),
+        orderBy("timestamp", "asc")
+      );
+    } else {
+      messageQuery = query(getLobbyChatCollection(lobbyData.id), orderBy("timestamp", "asc"));
+    }
+    unsubscribeChatMessages = onSnapshot(messageQuery, (collection) => {
+      chatMessages = collection.docs.map((message) => message.data());
+      console.log(collection.docs);
```

Comment:
prolly dont need this

Reply:
took it out and messages werent showing up

Reply:
What?

Reply:
are you sure you need the `console.log`?